test/json-stream-test.js

```diff
@@ -0,0 +1,23 @@
+var expect = require('chai').expect;
+var fs = require('fs');
+
+var JSONParseStream = require('../lib/streams/json-parse-stream');
+
+describe('JSONParseStream', function() {
+	it('should parse a json file into object', function(done) {
+		var readStream = fs.createReadStream('./test/resources/all_npm.json');
+		var jsonParseStream = new JSONParseStream();
+		readStream
+		.pipe(jsonParseStream)
+		.on('data', function (data){
```

Comment:
As noted above, this isn't the expected behavior of a JSON parse stream.

Also, using .on('data') will put the stream into streams1 mode, which we don't want, and may cause undesirable behavior.  zstreams provides internal utilities that make this kind of thing easy.